extensions/quarkus/runtime/src/main/java/org/apache/myfaces/core/extensions/quarkus/runtime/QuarkusFacesInitializer.java

```diff
@@ -36,28 +36,21 @@
  */
 public class QuarkusFacesInitializer extends FacesInitializerImpl
 {
-    private static boolean initialized = false;
     
     @Override
     public void initFaces(ServletContext servletContext)
     {
-        // DIRTY HACK as somehow quarkus initializes a second time?
```

Comment:
doesnt myfaces start 2 times with this change now? or did they fix it?

Reply:
I only see it start once with that warning about FactoryFinder?

Reply:
is public void initFaces(ServletContext servletContext) called 2 times?

Reply:
I am checking.

Reply:
OK it does initialize twice and again every time hot reload happens so my latest change make it work on initial startup and on hot reload with no more warnings!